ee-feature-pack/common/src/main/resources/license/ee-feature-pack-common-licenses.xml

```diff
@@ -826,6 +826,16 @@
         </license>
       </licenses>
     </dependency>
+    <dependency>
```

Comment:
Think we're missing the `opentelemetry-api-logs` dependency in here.

Reply:
Added.